routes/account.js

```diff
@@ -7,12 +7,16 @@ var Boom = require('boom')
 
 var errors = require('./utils/errors')
 var joiFailAction = require('./utils/joi-fail-action')
-var serialiseAccount = require('./utils/serialise-account')
+var serAccount = require('./utils/serialise-account')
+var serProfile = require('./utils/serialise-profile')
```

Comment:
We try to avoid abbreviations. Can we call these `serialiseAccount` /  `serialiseProfile` and below when binded with the options we could call them `serialiseAccountWithOptions` / `serialiseProfileWithOptions`?